scripts/extensions/sams/src/api/assets.ts

```diff
@@ -100,6 +100,17 @@ function querySetIds(source: IRootElasticQuery, params: IAssetSearchParams) {
     }
 }
 
+function queryAssetIds(source: IRootElasticQuery, params: IAssetSearchParams) {
+    if (params.assetIds != null && params.assetIds.length > 0) {
```

Comment:
It would be more meaningful to use a param name that indicates the Assets will be excluded (also if we later need a param to filter by Asset IDs)

Reply:
Agreed. May be `excludedAssetIds` ?